scripts/postcss/css-compile-variables.js

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const valueParser = require("postcss-value-parser");
+
+let aliasMap;
+let resolvedMap;
+let baseVariables;
+
+function getValueFromAlias(alias) {
+    const derivedVariable = aliasMap.get(`--${alias}`);
```

Comment:
Can we align on using variable names without the -- everywhere, and only prepend/trim them when going to/from postcss values? I feel like that might be a bit easier to follow if there are no double prepends/trims.

Reply:
like [this](https://github.com/vector-im/hydrogen-web/pull/701/commits/5d4323cd1dcabaacdfb98bb0b44ce09cc25a739d)?